fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -798,7 +814,7 @@ protected void process() {
      * @param groupName
      */
     protected void grantPrivileges(String groupName) {
-        if (!MULTITENANT_FEATURE_ENABLED.contains(dbType)) {
+        if (!PRIVILEGES_FEATURE_ENABLED.contains(dbType)) {
```

Comment:
if enable this command for postgresql, can we still use the Db2Adapter?

Reply:
Thank you for seeing this, I'll fix

Reply:
I've fixed this.